server/routes/key-worker-profile/update-capacity-status/routes.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response } from 'express'
+import { Services } from '../../../services'
+import { JourneyRouter } from '../../base/routes'
+import { UpdateCapacityAndStatusController } from './controller'
+
+export const UpdateCapacityAndStatusRoutes = ({ keyworkerApiService }: Services) => {
+  const { router, get, post } = JourneyRouter()
+  const controller = new UpdateCapacityAndStatusController(keyworkerApiService)
+
+  get('/', async (req: Request, res: Response) => {
+    const staffId = req.params['staffId'] as string
+
+    await controller.GET(req, res, staffId)
```

Comment:
```suggestion
  get('/', async (req: Request<{ staffId: string }>, res: Response) => {
    await controller.GET(req, res, req.params.staffId)
```
if we really need to get the staffId params and pass it to the GET controller here.

but i think JourneyRouter had `{ mergeParams: true }` which means the params would be passed to the next controller. 
`UpdateCapacityAndStatusController.GET` can get req.params.staffId directly, i don't think we need to pass the param here

Reply:
Moved the param call to the controller